### DISCUSSION TITLE
Task 2 LAB 2.04

### DISCUSSION THREAD

**Discussion by deivyjaguirre** [rramosp/ai4eng.v1#88] [02 - PYTHON]
Buenas noches, para ver si es posible que me colaboren con un problema que estoy teniendo en la tarea 2 del laboratorio 2.04, el código que tengo es el siguiente:
![image](https://user-images.githubusercontent.com/95886907/146482392-0c2f97d3-e47b-43ee-97f9-128d01974e96.png)

Si reviso lo que me arroja r por fuera de la función, es una lista reordenada, lo que sucede es que a la hora de evaluar me regresa "None":
![image](https://user-images.githubusercontent.com/95886907/146482623-dc27a4e7-b26e-43af-a8d3-85303c42d7fb.png)

No se que puede estar causando que no me arroje nada, ya que si saco el proceso de la función arroja la siguiente lista:
![image](https://user-images.githubusercontent.com/95886907/146482853-b600f28d-81df-4769-9ada-0ebe22eda0cc.png)

**Comment by Felipe-RA** [comment]
Saludos deivyjaguirre

No he podido probar tu código a fondo, la próxima vez adjunta tu notebook para poder revisarlo al completo.

Transcribí tu función pero obtuve un resultado distinto,  sospecho que por una implementación distinta de la función _make_heaps_ (no significa que la tuya sea necesariamente incorrecta, sólo incompatible con la que implementé yo)

![image](https://user-images.githubusercontent.com/49454068/146651693-0248e195-0bbf-4b68-a761-40ef4e5907af.png)


Para poderte ayudar comparteme tu notebook

Hasta pronto
